main.py

```diff
@@ -0,0 +1,10 @@
+import runner_class
+import run_race
+import asyncio
+
```

Comment:
Main code should be under
 if __name__ == '__main__':
           main()

def main(()
     ...............